packages/plugins/rubygem-http/rubygem-http.spec

```
@@ -0,0 +1,105 @@
+# Generated from http-4.1.1.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name http
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 4.1.1
+Release: 1%{?dist}
+Summary: HTTP should be easy
+Group: Development/Languages
+License: MIT
+URL: https://github.com/httprb/http
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby >= 2.3
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(http_parser.rb) >= 0.6.0
+Requires: %{?scl_prefix}rubygem(http_parser.rb) < 0.7
+Requires: %{?scl_prefix}rubygem(http-form_data) >= 2.0
+Requires: %{?scl_prefix}rubygem(http-form_data) < 3
+Requires: %{?scl_prefix}rubygem(http-cookie) >= 1.0
+Requires: %{?scl_prefix}rubygem(http-cookie) < 2
+Requires: %{?scl_prefix}rubygem(addressable) >= 2.3
+Requires: %{?scl_prefix}rubygem(addressable) < 3
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby >= 2.3
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end generated dependencies
+
+%description
+An easy-to-use client library for making requests from Ruby. It uses a simple
+method chaining system for building requests, similar to Python's Requests.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/.coveralls.yml
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.rubocop.yml
+%exclude %{gem_instdir}/.travis.yml
+%exclude %{gem_instdir}/.yardopts
+%{gem_instdir}/CHANGES.md
```

Maybe part of `%files doc`

```suggestion
%doc %{gem_instdir}/CHANGES.md
```